azure-pipelines.yml

```diff
@@ -25,7 +29,6 @@ jobs:
     workingDirectory: build_container
     env:
       OS_FAMILY: linux
-      IMAGE_ARCH: "amd64 arm64"
```

Comment:
are these supposed to be removed ?

Reply:
I went to update this so that only `amd64` was built for the mobile image, but they appear completely unused.

Reply:
The archs appear to be determined here: https://github.com/envoyproxy/envoy-build-tools/blob/5dde291ad6f64d8430b40d841a312bd8457eb526/build_container/docker_build_linux.sh#L19-L25

Reply:
yep - agreed

```console

$ git grep IMAGE_ARCH
azure-pipelines.yml:      IMAGE_ARCH: "amd64 arm64"
azure-pipelines.yml:      IMAGE_ARCH: "amd64"
build_container/docker_build_windows.sh:# Ignore IMAGE_ARCH as amd64 is the only option

```